### ISSUE TITLE
Bug: Error when installing integration

### ISSUE THREAD

**Issue by nielsfaber** [JurajNyiri/HomeAssistant-Tapo-Control#68]
I was trying to install your integration today.
I got stuck on this message:
![image](https://user-images.githubusercontent.com/16861605/107484908-5b34b200-6b83-11eb-99a4-3a8ed7da6fa7.png)

HA logs:
```
2021-02-10 09:33:27 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_protocol.py", line 422, in _handle_request
    resp = await self._request_handler(request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_app.py", line 499, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_middlewares.py", line 118, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 56, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 18, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 72, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 129, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 129, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 169, in post
    return await super().post(request, flow_id)
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 60, in wrapper
    result = await method(view, request, *args, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 106, in post
    result = await self._flow_mgr.async_configure(flow_id, data)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 155, in async_configure
    result = await self._async_handle_step(flow, cur_step["step_id"], user_input)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 213, in _async_handle_step
    result: Dict = await getattr(flow, method)(user_input)
  File "/config/custom_components/tapo_control/config_flow.py", line 71, in async_step_other_options
    enable_time_sync = user_input[ENABLE_TIME_SYNC]
KeyError: 'enable_time_sync'
```

Seems like a quick fix, by declaring a default value in the `user_input` dict or something.

After ticking the "Automatically synchronise time" box the issue is gone btw.

**Comment by JurajNyiri** [comment]
Fixed in 3.1.1, thank you for bug report! 👍

**Comment by nielsfaber** [comment]
Holy moly, you are fast as lightning!
I wish the HA dev team was this quick 😄 
Thanks a lot!
